hedgehog/src/Hedgehog/Internal/TH.hs

```diff
@@ -12,8 +12,8 @@ import qualified Data.Ord as Ord
 import           Hedgehog.Internal.Discovery
 import           Hedgehog.Internal.Property
 
-import           Language.Haskell.TH
-import           Language.Haskell.TH.Syntax
+import           Language.Haskell.TH (Exp (..), Q, TExp, location, runIO)
+import           Language.Haskell.TH.Syntax (Loc (..), mkName, unTypeQ, unsafeTExpCoerce)
```

Comment:
Just a minor nit, can you remove the space between `Exp` and `(..)` to match the style of the rest of the project (same for `Loc`)